// @okta/okta-angular/dist/index.js
"use strict";
/*
 * Copyright (c) 2017-Present, Okta, Inc. and/or its affiliates. All rights reserved.
 * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
 *
 * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
 * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 *
 * See the License for the specific language governing permissions and limitations under the License.
 */
Object.defineProperty(exports, "__esModule", { value: true });
var okta_module_1 = require("./okta/okta.module");
exports.OktaAuthModule = okta_module_1.OktaAuthModule;
var okta_guard_1 = require("./okta/okta.guard");
exports.OktaAuthGuard = okta_guard_1.OktaAuthGuard;
var okta_service_1 = require("./okta/services/okta.service");
exports.OktaAuthService = okta_service_1.OktaAuthService;
var okta_config_1 = require("./okta/models/okta.config");
exports.OKTA_CONFIG = okta_config_1.OKTA_CONFIG;
// Okta View Components
var components_1 = require("./okta/components");
exports.OktaCallbackComponent = components_1.OktaCallbackComponent;
exports.OktaLoginRedirectComponent = components_1.OktaLoginRedirectComponent;
